doc/classes/Control.xml

```diff
@@ -12,7 +12,7 @@
 		Call [method accept_event] so no other node receives the event. Once you accept an input, it becomes handled so [method Node._unhandled_input] will not process it.
 		Only one [Control] node can be in focus. Only the node in focus will receive events. To get the focus, call [method grab_focus]. [Control] nodes lose focus when another node grabs it, or if you hide the node in focus.
 		Sets [member mouse_filter] to [constant MOUSE_FILTER_IGNORE] to tell a [Control] node to ignore mouse or touch events. You'll need it if you place an icon on top of a button.
-		[Theme] resources change the Control's appearance. If you change the [Theme] on a [Control] node, it affects all of its children. To override some of the theme's parameters, call one of the [code]add_theme_*_override[/code] methods, like [method add_theme_font_override]. You can override the theme with the Inspector.
+		[Theme] resources change the [Control]'s appearance. The [member theme] of a [Control] node affects all of its direct and indirect children (as long as a chain of [Control]s is uninterrupted). To override some of the theme items, call one of the [code]add_theme_*_override[/code] methods, like [method add_theme_font_override].
```

Comment:
```suggestion
		[Theme] resources change the control's appearance. The [member theme] of a [Control] node affects all of its direct and indirect children (as long as a chain of controls is uninterrupted). To override some of the theme items, call one of the [code]add_theme_*_override[/code] methods, like [method add_theme_font_override].
```
I think this is a little better personally, but since the salvaged PR already had an applied suggestion to use "[Control]" instead of "control" I left it in (and added brackets to the untagged capitalized "Control").

Reply:
Doing this has always been better in my opinion, yes.

Reply:
Applied this change, but willing to revert if it's contentious.

We should probably formalize some actual guidelines for when to use "control" vs "[Control]" at some point, otherwise in ambiguous situations like this are likely to be changed back and forth repeatedly.